python/paddle_fl/mpc/data_utils/aby3.py

```diff
@@ -345,6 +345,31 @@ def _transpile_type_and_shape(block):
             elif op.type in op_to_skip:
                 pass
             else:
+                if op.type == 'pool2d':
```

Comment:
The logic is OK, but putting the stuff here will make the function not neat and the code hard to maintain when more op need the special care.

Considering replace them with a method: 
add_extra_op_desc(op)
And use a ABC call OpExtraDesc, derivate it as Pool2dExtraDesc, ReluExtraDesc, etc.
In OpExtraDesc, add an abstract method called 'add_extra_desc(op_desc)', which contains the extra stuffs here.

Reply:
done